contract/src/start-game1-proposal.js

```diff
@@ -18,75 +18,89 @@ const CENT = IST_UNIT / 100n;
 /**
  * Make a storage node for auxilliary data for a value on the board.
  *
- * @param {ERef<StorageNode>} chainStorage
- * @param {string} boardId
+ * @param {ERef<StorageNode>} chainStorage - Reference to the chain storage.
```

Comment:
Is "Reference to the chain storage" sufficiently more clear than just `chainStorage` to take on the maintenance burden?

Reply:
Understood